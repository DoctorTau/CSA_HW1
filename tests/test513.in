581
8209 -7070 4443 2943 7546 -3168 -5782 -6333 -9330 -7072 -8590 -550 7744 3702 -5908 7323 -4313 -9399 -689 -807 -8781 6993 1477 8564 -4780 -2613 8522 -7974 1617 -5909 -96 -6033 3245 -5050 8631 5400 6384 -5628 1311 -228 -9424 -374 7667 7751 -2364 -8146 4811 -4621 -2995 -2773 9551 6979 -5604 -9628 4829 7507 4833 3799 1583 2398 9035 -3936 -623 -6715 -2191 -3228 2171 -4540 2178 -5826 74 5036 5912 9427 -8746 -6569 6057 5401 -2102 -512 -2862 5007 -1938 -3622 9243 -5040 9446 8582 8386 -7527 933 -1107 6241 2251 5224 -9026 8044 6038 865 2177 1996 -4418 7125 -2344 7741 4620 6982 9736 -8386 -3216 -2524 -6721 -4923 -9671 8074 2024 -5267 2683 -4994 -4049 -6086 -4360 -3360 -5908 7553 5222 5972 -685 -7213 5946 8594 9352 5966 1551 2621 -6765 -8219 -1033 -8115 925 -1066 1838 7635 -3892 399 -6404 2618 2962 -4974 -6958 1781 297 -7862 -43 -3412 4224 -115 508 49 1959 5035 2996 -4622 9463 7824 -3107 6593 -898 -9096 3081 -6487 6435 -1930 -9538 -4642 -1272 -9369 -4442 334 4879 -879 8649 -9161 -6870 6302 -2507 -6023 5796 7490 -8478 8650 -9984 -8698 1399 -4770 -1899 -9120 1802 6567 4993 -2274 -9576 -614 -4188 4427 -1658 4487 3659 -8937 909 -3363 504 2028 -9053 5162 4750 1123 -7857 -6900 3785 6104 3684 -285 1749 -8902 -6812 -1501 4874 -1202 9696 -4583 -8179 -6434 6283 4924 -2599 -4465 9560 -9477 3081 7352 7521 -8888 -2636 5135 -3978 -523 -5205 5391 -9544 -3521 -4111 -4267 3008 -8701 1916 8388 -9999 -7931 6659 5620 -208 9028 -594 -5051 -7047 8033 5695 -1442 5777 -5437 3059 3191 -563 -4314 9991 2047 -3609 5299 3216 5930 -7785 4785 4405 -7537 -6233 1829 8192 -4938 846 -9899 -6804 5924 5878 -8859 278 -4513 2675 1289 -6264 2393 2101 2591 451 -7143 -6972 4382 -2369 8902 5832 6548 7268 -5439 4195 -7786 6004 1475 524 -4740 4142 -6853 -3091 3337 3205 -8207 8714 -8257 -7923 -8602 2514 -6043 5228 9828 -8150 -1142 -1833 890 4040 665 -1892 2141 3620 -1952 -9597 6079 -7693 6121 5363 -578 -6125 5269 8609 -2056 1161 -4710 -667 -2185 4455 -8399 4301 3191 -3064 1583 -7267 -2085 2932 -5919 -5362 -7246 8231 6232 920 3270 -1076 -1907 -1147 4611 -3330 7516 7278 796 -2106 -3626 8664 2792 -9692 -1042 -4040 6648 -738 -8598 3111 7817 -3373 8589 965 -514 3926 -6204 1835 5351 7079 -5027 -4713 -5914 8613 -9212 9191 329 -9547 1715 -3635 -3246 -2546 -3409 3202 -123 6830 -9480 -7218 -6438 -1210 -562 7157 -5323 -8443 94 -3388 6399 -2199 5392 -3935 -4337 -5765 -9182 1960 9669 7008 7281 -2445 8677 435 9985 -4324 2027 -7137 -776 -7828 -541 1392 -7142 8131 3023 -1064 -3082 9425 -3703 -4829 -703 -7060 -3269 2552 1403 8388 -3813 -7327 -4106 -7177 4862 -6883 6194 -1185 -6750 -4559 5126 -4063 4269 -8114 5608 -4986 6222 9012 2716 -4808 4664 9401 -8391 -9928 -8815 -8252 1360 5881 -8596 -7674 -4775 4405 -9569 -4400 4136 3429 -511 -7942 7106 9691 -1464 -8014 -2213 4489 8218 -8704 -5301 6211 -4094 4730 9461 -3872 1959 -2353 -1485 -2442 -3556 -1708 759 4924 9051 433 4054 8361 5750 -9758 -984 -7302 7431 -7206 3442 3932 -1063 -4741 -8015 343 -545 4734 -117 5347 -9912 1756 -5076 -1744 -9970 -1985 3247 -2268 4704 1022 -675 -1913 -8946 -5500 -5464 -3608 -7452 1225 5216 -3808 6264 -8066 7421 -6268 -5341 -7229 6823 847 8504 -6692 -9584 -8535 7406 -1898 6504 -3175 9451 