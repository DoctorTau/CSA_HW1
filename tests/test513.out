-7070 -3168 -5782 -9330 -7072 -8590 -550 -5908 -4780 -7974 -96 -5050 -5628 -228 -9424 -374 -2364 -8146 -5604 -9628 -3936 -3228 -4540 -5826 -8746 -2102 -512 -2862 -1938 -3622 -5040 -9026 -4418 -2344 -8386 -3216 -2524 -4994 -6086 -4360 -3360 -5908 -1066 -3892 -6404 -4974 -6958 -7862 -3412 -4622 -898 -9096 -1930 -9538 -4642 -1272 -4442 -6870 -8478 -9984 -8698 -4770 -9120 -2274 -9576 -614 -4188 -1658 -6900 -8902 -6812 -1202 -6434 -8888 -2636 -3978 -9544 -208 -594 -1442 -4314 -4938 -6804 -6264 -6972 -7786 -4740 -8602 -8150 -1142 -1892 -1952 -578 -2056 -4710 -3064 -5362 -7246 -1076 -3330 -2106 -3626 -9692 -1042 -4040 -738 -8598 -514 -6204 -5914 -9212 -3246 -2546 -9480 -7218 -6438 -1210 -562 -3388 -9182 -4324 -776 -7828 -7142 -1064 -3082 -7060 -4106 -6750 -8114 -4986 -4808 -9928 -8252 -8596 -7674 -4400 -7942 -1464 -8014 -8704 -4094 -3872 -2442 -3556 -1708 -9758 -984 -7302 -7206 -9912 -5076 -1744 -9970 -2268 -8946 -5500 -5464 -3608 -7452 -3808 -8066 -6268 -6692 -9584 -1898 